ern-api-gen/src/apigen.js

```diff
@@ -103,24 +85,40 @@ export async function regenerateCode (options: Object = {}) {
   })
 
   await cleanGenerated()
+
+  const pkgConfig = {
+    npmScope : pkgName && pkgName.startsWith('@') ? pkgName.slice(1, pkgName.indexOf('/')) : '',
+    moduleName : pkgName &&  pkgName.startsWith('@') ? pkgName.slice(pkgName.indexOf('/') + 1) : pkgName,
+    apiVersion : isNewVersion ? newPluginVer : pkg.version,
+    apiDescription : pkg.description,
+    apiAuthor : pkg.author,
+    bridgeVersion : options.bridgeVersion,
+    ...extra
+  }
+  // Regenerate package.json
+  fileUtils.writeFile(`${process.cwd()}/${PKG_FILE}`, generatePackageJson(pkgConfig))
+  // Regenerate .flowconfig file
+  fileUtils.writeFile(`${process.cwd()}/${FLOW_CONFIG_FILE}`, generateFlowConfig())
+
```

Comment:
For these two lines please make use of `path.join` instead of building the path manually, for better cross platform support. 

instead of 

```javascript
`${process.cwd()}/${PKG_FILE}` 
``` 

use 

```javascript
path.join(process.cwd(), PKG_FILE)
```

Reply:
true. should have noticed that, that's how we are adapting code lately.